hw/formal/data/dedicated_fpv_cfgs.hjson

```diff
@@ -0,0 +1,74 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+// This hjson file list all the IPs that are fully verified by FPV
+{
+  use_cfgs: [
+             {
+               name: prim_arbiter_ppc_fpv
+               fusesoc_core: lowrisc:fpv:prim_arbiter_ppc_fpv
+               import_cfgs: ["{proj_root}/hw/formal/data/common_fpv_cfg.hjson"]
+               rel_path: "hw/ip/prim/fpv"
```

Comment:
Try adding `tool: jaspergold` so that `-t jaspergold` doesn't have to be specified on the command line.

Reply:
That is a good idea, but seems like `OneShotCfg.py` does not allow the argument `-t` to be empty (https://github.com/lowRISC/opentitan/blob/master/util/dvsim/OneShotCfg.py#L26). I will work with Michael to see if he is also in flavor of removing this constraint and add a default tool to each flow.